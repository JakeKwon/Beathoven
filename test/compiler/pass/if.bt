func if_else_tester(bool flag) -> unit
{
	if (flag) {
		print(true);
	}
	else {
		print(false);
	}
	print(0, "\n");
}
if_else_tester(true);
if_else_tester(false);
