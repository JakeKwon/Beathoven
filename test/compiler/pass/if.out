10
00

1